mmdet3d/models/necks/pointnet2_fp_neck.py

```diff
@@ -0,0 +1,63 @@
+from mmcv.runner import BaseModule
+from torch import nn as nn
+
+from mmdet3d.ops import PointFPModule
+from mmdet.models import NECKS
+
+
+@NECKS.register_module()
+class PointNetFPNeck(BaseModule):
+    r"""PointNet FP Module used in PointRCNN.
+
+    Refer to the `official code <https://github.com/charlesq34/pointnet2>`_.
+
```

Comment:
may draw a code block here

Reply:
https://github.com/open-mmlab/mmdetection/blob/master/mmdet/models/roi_heads/mask_heads/fused_semantic_head.py#L15

Reply:
Use code-block, otherwise they cannot show properly in the documentation

Reply:
https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/core/bbox/structures/cam_box3d.py#L14